charts/kong/values.yaml

```diff
@@ -183,8 +183,18 @@ plugins: {}
 # - kong-admin-tls
 secretVolumes: []
 
-# Set runMigrations to run Kong migrations
-runMigrations: true
+# Enable/disable migration jobs, and set annotations for them
+migrations:
```

Comment:
Question: Do we need to clarify with a doc line saying this is only valid for Kong installations backed by DB?
The reason I ask is because it is not totally obvious from just reading this.

Reply:
Not really--the job templates already disable themselves if DB-less is in use, and no changes are required to this config for it.